scripts/userstyles.yml

```
@@ -416,6 +416,13 @@ userstyles:
     readme:
       app-link: "https://news.ycombinator.com"
     current-maintainers: [*lucasmelin]
+  haveibeenpwned:
```

Probably should do proper kebab case right?
```suggestion
  have-i-been-pwned:
```

---

i suppose

---

Make sure to update the folder name, userstyles.yml metadata, and usercss metadata.